src/types.js

```diff
@@ -59,6 +62,11 @@ export type SiteAdapter = {
     slug: string,
     url: string,
     title: string,
+    description: ?string,
+    // authors: Array<{ name: string, role: 'art' | 'story' | 'unknown' }>,
+    author: ?string,
```

Comment:
Ahh, another tricky case here: when it's the same person for both story and art, what do we return?

```jsx
[{ name: 'David', role: 'story' }, { name: 'David', role: 'art' }]
// or something like:
[{ name: 'David', role: 'all' }]
```

Returning two things might be confusing for anyone who tries to use `series.authors.length`

Reply:
After implementing the Array approach, I'm less a fan of it. The case mentioned above, where a single person handles everything, is actually quite common. It feels weird to make that the default behaviour since it doesn't have great ergonomics.

Lets imagine I'm trying to build out a React component that uses this information:

```jsx
const SeriesSummary = ({ series }) => (
  <div>
    <p>{series.description}</p>
    <ul>
      {series.authors.map(author => (
        <li>{author.role}: {author.name}</li>
      )}
    </ul>
  </div>
);
```

But there's more issues here.
* You need to worry about the ordering: does "Story" come first, or does "Art" come first?
* You now have to deal with properly labelling the roles. Maybe you map it out like this:

```jsx
const RoleLabels = {
  story: 'Author',
  art: 'Artist',
  unknown: 'Other'
};

// Then use it like this...
<li>{RoleLabels[author.role]}: {author.name}</li>
```

* You need to deal with grouping roles

```jsx
series.authors
// => [{ name: 'David', role: 'story' }, { name: 'Alex', role: 'art' }, { name: 'Ellen', role: 'story' }]

// Would result in
<li>Author: David</li>
<li>Artist: Alex</li>
<li>Author: Ellen</li>
```

Since I've yet to see any other roles beyond "story" and "art", I'm inclined to keep those two simple fields for now, `author` and `artist`. It'd be nice if there was a way to simplify it if there was only one name. But until there's a good use-case for something more complex, I'll keep it suuuuper simple.

Reply:
I've opted to go with a single `author` field for now. It'll contain either `null` or a string of all the authors formatted into a human-readable list, automatically de-duping any values:

```jsx
series.author //=> "MIZUSHINA Takayuki"
series.author //=> "HYUUGA Natsu, NANAO Ikki, Nekokurage"
```

This work was done in https://github.com/poketo/node/pull/30/commits/c9159a78f298733fb7eb4ebcb5695d0183787264